spec/fixtures/stack_master.yml

```diff
@@ -4,6 +4,8 @@ region_aliases:
 stack_defaults:
   tags:
     application: my-awesome-blog
+template_compilers:
+  rb: ruby_dsl
```

Comment:
This was put in solely as test data (and intentionally avoided any connection with cfndsl, as this could be anything and the code we test wouldn't care about it). Should we use cfndsl instead?